packages/paste-core/components/list/src/index.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import {Text, NULLIFY_TEXT_PROPS} from '@twilio-paste/text';
+
+interface BaseListProps extends React.OlHTMLAttributes<''> {
+  className?: never;
+  style?: never;
+  as: 'ol' | 'ul';
+}
+
+const List: React.FC<BaseListProps> = ({as, children, ...props}) => {
+  return (
+    <Text
+      {...props}
+      {...NULLIFY_TEXT_PROPS}
+      as={as}
+      marginBottom="space70"
+      marginLeft="space70"
+      fontSize="fontSize30"
+      lineHeight="lineHeight30"
+      fontWeight="fontWeightNormal"
+      textColor="colorText"
+    >
+      {children}
+    </Text>
+  );
+};
+List.displayName = 'List';
+
+export type OrderedListProps = Omit<BaseListProps, 'as'>;
+const OrderedList: React.FC<OrderedListProps> = ({children, ...props}) => {
+  return (
+    <List {...props} as="ol">
+      {children}
+    </List>
+  );
+};
+OrderedList.displayName = 'OrderedList';
+
+export type UnorderedListProps = Omit<BaseListProps, 'as'>;
+const UnorderedList: React.FC<UnorderedListProps> = ({children, ...props}) => {
+  return (
+    <List {...props} as="ul">
+      {children}
+    </List>
+  );
+};
+UnorderedList.displayName = 'UnorderedList';
+
+export interface ListItemProps extends React.LiHTMLAttributes<''> {
```

Comment:
Same deal, legit typed as an `<li />` element, totally safe to pass any prop to. Styles won't change